### ISSUE TITLE
Files don't download

### ISSUE THREAD

**Issue by GormCh** [gmansillo/simple#119]
I'm building a site with Joomla 3.9. I need to link to many documents, mostly pdf, for site visitors to download. Our old site is based on J1.5 where I used the Weblink feature. 

For the new site I tried the standard Joomla Document Manager. I created a Category and a  couple of files for it. The files showed up as expected on the Front page, with download buttons. 

When clicking one of the front page download button the normal download popup appeared immediately followed by a new popup informing that "the file could not be saved because the source file could not be read". I tried different browsers to exclude that part and also different computers. I also downloaded the true Firedrive instead of the standard version Simple. The only difference in behaviour was that the builtin pdf-reader in Firefox started but with no pdf file loaded. Same goes files like pdf, xpi, docs. Image files like jpg, bmp and png show up as images on the screen when clicking the Download.

My question is. Are there any specific settings needed to be activated or set in Joomla 3.9 to get Firedrive to work with other files than image files? 

Thanks
Gorm Chandler

**Comment by gmansillo** [comment]
Hi @GormCh,
sorry for delay in reply. Do you upload files from frontend or from backend?

**Comment by GormCh** [comment]
Hi there and thanks for coming back.

It was from the Backend. I don't know what caused it. I moved the files and 
the database to a new installation and it  works fine.i use Firedrive now.

I have though a request. Could you add an Open button next to the Download 
button. If it is a pdf a click or a button press could then open the 
pdf out an image directly in the browser.

I think the app is very good, clean and with no fuzz.

I think that it came with some Swedish translation but it wasn't completed. 
I will do that and can send you the file of you're interested.

Gorm

Den 25 februari 2019 22:34:50 skrev Giovanni Mansillo 
<notifications@github.com>:

**Comment by GormCh** [comment]
Hi!
The download problem still exist. The files lists as they should on the frontpage with download buttons. Clicking download result in a error popup saying something like .part  can not be saved because the source file could not be read.

It does not matter which browser is used. Any ideas?

Gorm

**Comment by alkemica** [comment]
Hi, i have the same problem: "Clicking download result in a error popup saying something like .part can not be saved because the source file could not be read.". 
I try to upload any type of file and the problem is with all file up to 200 kb.

**Comment by GormCh** [comment]
Hi
I guess that it works with txt and image files.
I eventually got it working by setting SEF to Yes in Global Systems settings, activating the SEF component and also activating .htaccess, i.e. re-naming htaccess.txt to .htaccess. I have the same problem on another site which has been moved top around in different installations. I didn't manage to fix it. I uin-installed and installed Phoca whcih worked there.
Gorm
